docs/stdlib/numbers.rst

```diff
@@ -644,12 +663,11 @@ from :eql:type:`str` and :eql:type:`json`.
                   std::bit_lshift(val: int32, n: int64) -> int32
                   std::bit_lshift(val: int64, n: int64) -> int64
 
-    Bitwise left-shift operator for intergers.
-
-    The integer *val* is shifted by *n* bits to the left. The rightmost added
-    bits are all ``0``. Shifting an integer by a number of bits larger than
-    the bit size of the integer results in ``0``.
+    Returns the result of a bitwise left-shift operator for two integers.
```

Comment:
```suggestion
    Returns the result of a bitwise left-shift operation on an integer.
```

Even though there are two integers passed, it doesn't make sense to characterize this as an operation "for two integers" since the second only defines the number of bits to shift.